client/app/components/SearchBar/SearchBar.js

```diff
@@ -13,16 +15,139 @@ const SEARCH_MODES = [
   SEARCH_MODE_KEYWORD_AND_LOCATION,
 ];
 
+const coordinates = (place) => {
+  if (place && place.geometry) {
+    return place.geometry.location.toUrlValue();
+  }
+  return null;
+};
+
+const getDetails = (placeId) => new Promise((resolve, reject) => {
+  const serviceStatus = window.google.maps.places.PlacesServiceStatus;
+  const div = document.createElement('div');
+  const service = new window.google.maps.places.PlacesService(div);
```

Comment:
@mporkola [Weird API](https://developers.google.com/maps/documentation/javascript/reference#PlacesService)

The `div` element isn't used anywhere, but API just requires one.